README.md

```diff
@@ -1,2 +1,102 @@
 # leaderboard-server
-A leaderboard service for Mozilla Geolocation Stumbling
+
+The leaderboard-server is a RESTful service which implements an API
+for submitting data which describes the geolocation stumbling efforts
+of contributors to the Mozilla Location Service project.  Mozilla Stumbler
+users can submit data to the leaderboard-server about how many networks
+were detected within a geospatial 'tile' at a given time, and then query
+the top contributors.
+
+# API Interface
+
+Add Stumbles
+----
+ Submit the number of networks detected within a series of tiles.
+
+* **URL**
+
+  https://leaderboard.services.mozilla.com/backend/add_stumbles
+
+* **Method:**
+
+  `POST`
+
+*  **URL Params**
+
+  None
+
+* **Data Params**
+
+  POST body (JSON encoded)
+
+        {
+          items: [
+            {
+              tile_east: <int>,
```

Comment:
We should add a note that the "tile" refers not to the server-side tiles, it is for client-side tiling to aggregate observations and represents a point around which a rectangular tile was used to aggregate observations, so that the client doesn't need to report 1:1 coordinate and observation counts.
We are following the convention that the size of the tile on the client is a 1km tile. We recommend all apps follow that same tile size convention, as it correlates then with the tile size on the server. There is not absolute requirement for this. A smaller tile means necessary extra rows in the JSON file, a larger tile means that observations are being stored less accurately, and the error level of polygonal geo queries increases.